job-descriptions/engineering/engineering-manager-campaigns.md

```diff
@@ -0,0 +1,47 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Engineering Manager - Campaigns
+
+We are looking for an experienced engineering leader to manage our cross-functional engineering team building [campaigns](https://about.sourcegraph.com/handbook/engineering/campaigns). You will partner with our [product team](https://about.sourcegraph.com/handbook/product) to define team goals, and develop a roadmap to achieve those goals. Our product and team is very technical; we expect you to have a strong background as an engineer so that you can contribute to technical discussions and contribute code when necessary/appropriate. The team is small today but there is a lot more work to be done and we will expect you to grow the team in a sustainable way that maintains our high standards.
+
```

Comment:
What about general traits of the person? Off the top of my head, things like these:

* They value transparency & honesty.
* They value communication skills and are an excellent communicator.
* They don't insist on process for process' sake.
* They wish that we could work without "politics" but also accept that in certain situations that's inevitable and know how to play the game if they have to.
* They see the team members as fellow team members, not as "my" employees/team members.
* They ensure that team members have enough freedom & autonomy to do their best work.
* They trust and empower team members to do their best work.
* They know when the best way to help someone is to get out of their way and when it requires jumping in and actively helping out.
* They are sensitive to others and are able to "read the room", even and especially in written conversation
* They are humble enough to admit when they don't know something.
* They seek out feedback because the have a "growth mindset" and know how to give constructive feedback.
* They have really high standards and willing to push for them — in technical work, leadership, communication.
* They try to create an environment of positivity, creativity and trust.

I know some of that sounds like too much fluff, but: top of my head. We can reformulate that :)

Reply:
Another thought: it would be great if the manager was technical enough to also do code reviews and review RFCs on their technical feasibility.

Reply:
I'd think those are necessary skills for the eng manager, no? Not just "it would be great" things.

Reply:
In my opinion: yes. But I'm not 100% sure what the desired state is with eng managers, project managers, project leads, etc. For example, I could also imagine a possible scenario in which a manager, next to a project/tech lead, is more in a coach-y role. So that's why I prefixed it. But I think here we do want a more technical manager.

Reply:
> In my opinion

Your opinion is the most important opinion when it comes to hiring a manager for the campaigns team :)

Yes we want a manger who can do code reviews and review RFCs. This is not mutually exclusive with having strong technical leaders on the team who aren't people managers (like yourself).

The term "project lead" was created before we decided to move in the direction of having an engineering manager per mission team, so I expect that name and definition will evolve.

Reply:
Alright, then it goes in.

@nicksnyder what are the next steps with this description? I'd love to share it. I'm also happy to share an open PR (gives us some street cred 😎 ) but having a final PR without suggestions is a better signal to candidates, because then it's clear that we know what we want.